schemas/resources/schemas/order-option/v1.0.0/schema.json

```diff
@@ -7,18 +7,17 @@
   "properties": {
     "Id": {
       "description": "The identifier that was assigned to this record by an older system. It is in the schema to preserve search API functionality in order to store this identifier.",
-      "type": "string",
-      "maxLength": 200
+      "type": "string"
```

Comment:
Why do we remove the maxLength from the id field? I think it would be good to have a maxLength.

Reply:
Service option schema doesn't have a length associated with the ID field in the rest API.  Do you know if service option ids are stored in the same DB table as order options?  I can add the maxLength back in, I just don't want to eliminate existing data.

Reply:
The oracle db schema has maxLength of 200 on the guid field of both tables, so maxLength of 200 will be safe.

Reply:
maxLength added back in.